### PR TITLE
refactored chart ui display format

### PR DESCRIPTION
**Pull request by tobianointing**
- refactored chart ui display format
- created sort functins to sort the github and twitter data according to hour,weekdays,day

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Asalytics
+# Nazari
 
-Asalytics Public Link [asalytics.ai](https://asalytics.ai)
+Nazari Public Link [asalytics.ai](https://asalytics.ai)
```
Comment (OladetounJed) [inline README.md:3]: Change asalytics.ai to nazari.ai

File: pages/[asaId]/github/index.tsx [hunk 2]
```diff
@@ -3,31 +3,40 @@ import { useEffect, useState } from "react";
...
+    const mostCommitsWeekday = getMostDoneInWeekDay(sortedData, "commits")?.lastPushDateWeekday ?? "";
+    const mostIssuesWeekday = getMostDoneInWeekDay(sortedData, "issues")?.lastPushDateWeekday ?? "";
+
     const [commitAnalyticsState, setcommitAnalyticsState] = useState([] as any);
     const [issueAnalyticsState, setissueAnalyticsState] = useState([] as any);
 
     let commitAnalytics = [] as Array<any>;
     let issueAnalytics = [] as Array<any>;
 
+    console.log(commitAnalytics);
```
Comment (OladetounJed) [inline pages/[asaId]/github/index.tsx:35]: Remove the console.log

**Review by OladetounJed — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tobianointing) [reply README.md]: fixed

File: pages/[asaId]/github/index.tsx (hunk 2 above)
Comment (tobianointing) [reply pages/[asaId]/github/index.tsx]: fixed

**Review by OladetounJed — APPROVED** [review]

**Pull request merged**